types/node/test/url.ts

```diff
@@ -0,0 +1,153 @@
+import assert = require("assert");
```

Comment:
I had to move the tests here to workaround https://github.com/DefinitelyTyped/DefinitelyTyped/issues/40967